### ISSUE TITLE
So many functions

### ISSUE THREAD

**Issue by cpeck-rmvd** [starlingcode/viaflash#12]
In via resource center,

Why is there so many defined functions to handle this `    def on_slot1_clicked(self):`? Are they referenced later?

In any case, a loop seems optimal whether just to execute the code in the functions or create the functions themselves.

**Comment by liquidcitymotors** [comment]
Yeah this does kinda suck! A more elegant strategy can be found here https://github.com/starlingcode/viaflash/blob/e5cbe1258a6c09d6942fe8bb42b73bc5772182da/wavetable_set_editor.py#L272. Shouldn't be too much to rework this and it will make the source a lot easier to read.
